### PR TITLE
Add `million@2.0.0` benchmark

### PR DESCRIPTION
**Pull request by aidenybai**
Removes `unkeyed` Million 1.x implementations. 

Adds Million 2.x `keyed` implementation with (#801)

### CODE REVIEW

File: frameworks/keyed/million/src/main.js [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    );
+  };
+  return (props, key) => {
+    return new Block(root, edits, props, key, shouldUpdate);
+  };
+})();
+
+(() => {
+  new Block(
+    stringToDOM(
```
Comment (aidenybai) [inline frameworks/keyed/million/src/main.js:216]: Does this count as too much optimization here?

Comment (fabiospampinato) [reply frameworks/keyed/million/src/main.js:216]: Is this code generated by a compiler? (Asking since it's under /src)

Comment (aidenybai) [reply frameworks/keyed/million/src/main.js:216]: No, it's not. I can rewrite it such that it uses the canonical API instead, do you think that's more aligned with this bench?

Comment (fabiospampinato) [reply frameworks/keyed/million/src/main.js:216]: Yeah that would probably be better, I can't imagine people in practice will write code like that.

Comment (aidenybai) [reply frameworks/keyed/million/src/main.js:216]: Got it, thanks!

**Comment by krausest** [comment]
No doubt, it's pretty fast, but looking at the code I'm sure we'll have lots of discussions what flags to add (I propose #772). 
Bildschirm­foto 2023-03-15 um 22 15 00

**Comment by ryansolid** [comment]
I see.. hmm... @aidenybai you can correct me, but it looks like on select you look up the VDOM nodes and then patch them directly. I agree this verges on #772. Instead of writing code to describe the selection in the view you do an imperative lookup and mutation.

**Comment by aidenybai** [comment]
Sounds good, I agree with the #772 flag. Is the best way to add a new flag to open up a PR?

**Pull request merged**